src/Microsoft.DotNet.Tools.Builder/ProjectDependenciesFacade.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Compilation;
+using Microsoft.DotNet.Tools.Compiler;
+using Microsoft.DotNet.ProjectModel.Utilities;
+
+
+namespace Microsoft.DotNet.Tools.Build
+{
+    //facade over the dependencies of a project context
+    internal class ProjectDependenciesFacade
+    {
+        //projectName -> ProjectDescription
+        public Dictionary<string, ProjectDescription> ProjectDependenciesWithSources { get; }
+        public List<LibraryExport> Dependencies { get; }
+
+        public ProjectDependenciesFacade(ProjectContext _rootProject, string configValue)
```

Comment:
I rewrote the way that class got instantiated a few times. A field name must have slipped in. Thanks for pointing that out.